dynamoDbQueueSource.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var _ = require('lodash');
+var AWS = require('aws-sdk');
+
+var dynamoDb = new AWS.DynamoDB();
+
+var dynamoDbQueueSource = { };
```

Comment:
Lots of gaps in objects everywhere. I feel like we already settled this in our PHP, but it's back.